hangups/default.conf

```diff
@@ -0,0 +1,11 @@
+[General]
+col-scheme: default
+
+[Keybindings]
+key-prev-tab: ctrl u
+key-next-tab: ctrl d
+key-close-tab: ctrl w
+key-quit: ctrl e
+key-menu: ctrl n
+key-up: k
+key-down: j
```

Comment:
I see your point. I was thinking it could act as a global default, but that's not useful at all. I'll move it to the root of the repo. I'll also add the default datetime formats, since they've now been merged into main.